litmus-portal/graphql-server/pkg/database/mongodb/operations/project.go

```diff
@@ -98,3 +111,32 @@ func UpdateInvite(ctx context.Context, projectID, userName string, invitation db
 	}
 	return nil
 }
+
+//CancelInvite ...
+func CancelInvite(ctx context.Context, projectID string, userName string) error {
+	query := bson.M{"_id": projectID}
+	update := bson.M{"$pull": bson.M{"members": bson.M{"username": userName}}}
+	_, err := projectCollection.UpdateOne(ctx, query, update)
+	if err != nil {
+		log.Print("Error Canceling the invite of member with username:", userName, "from project with project id: ", projectID, err)
+		return err
+	}
+	return nil
+}
+
+//UpdateRole ...
+func UpdateRole(ctx context.Context, projectID string, userName string, Role *model.MemberRole, invitation dbSchema.Invitation) error {
```

Comment:
Can't we use the `updateInvite` function itself by passing one optional parameter i.e `Role`